extensions/exposition/source/HTTP/Context.ts

```diff
@@ -27,6 +28,11 @@ export class Context {
     this.timing = new Timing(properties.trace)
     this.debug = properties.debug
 
+    if (this.debug) {
+      console.debug(`${request.method} ${this.url.href}`)
+      logHeaders(request.headers)
```

Comment:
1. It would be impossible to parse this multiple lines output as a single log entry.
2. I'm actually not sure if `console.debug` provides concurrency safety. It is synchronous, whereas writing anything anywhere is asynchronous by its nature.